build/cmake/locale/CMakeLists.txt

```diff
@@ -1,16 +1,20 @@
-if(wxBUILD_LOCALES STREQUAL "AUTO")
-    find_package(Gettext QUIET)
-elseif(wxBUILD_LOCALES)
-    find_package(Gettext REQUIRED)
+if(wxBUILD_LOCALES)
+    find_package(Gettext)
```

Comment:
Why did you remove `QUIET` in the `AUTO` option? Now it pollutes the output every reconfigure with
```
Could NOT find Gettext (missing: GETTEXT_MSGMERGE_EXECUTABLE GETTEXT_MSGFMT_EXECUTABLE)
```

Reply:
Sorry, I wanted to simplify the code but didn't realize that it would start giving warnings in AUTO case when it's not available (I've only tested this here, where it is).

I've restored it now, I don't think using a temp variable and passing it to `find_package()` is better than doing it with just an `if()`, is it?